server/src/lib/utils/validator.ts

```diff
@@ -158,7 +158,9 @@ function validateAuthor(author: any, errors: Array<ajv.ErrorObject>) {
 function validateHeroImage(heroImage: string, errors: Array<ajv.ErrorObject>) {
   const constraints = {
     website: {
-      url: true
+      url: {
+        allowLocal: true
+      }
```

Comment:
@SarvaBharan Can you write a small test for this ?

Reply:
Added testcases